tests/unit/services/test_generic_proxy.py

```diff
@@ -1,272 +1,7 @@
 import pytest
 
 from localstack import config
-from localstack.services.generic_proxy import UrlMatchingForwarder, get_cert_pem_file_path
```

Comment:
This was removed because the UrlMatchingForwarder was not used anymore except by these tests. But I wasn't sure, since deleting this many tests is a rather "destructive" move

Reply:
That's totally fine, there shouldn't be hard feelings when deleting old code. And the tests just belong to it. ;)